scripts/scheming_migration/index.js

```diff
@@ -210,12 +222,15 @@ async function main() {
 				}
 				resource['extras']['temporal_extent'] = JSON.stringify(temporalExtent);
 				if (proj_name[resource['extras']['projection_name']]) resource['extras']['projection_name'] = proj_name[resource['extras']['projection_name']];
-				if (resource['extras']['resource_storage_access_method']) resource['extras']['resource_storage_access_method'] = resource['extras']['resource_storage_access_method'].toLowerCase();
+				if (resource['extras']['resource_storage_access_method']) resource['extras']['resource_access_method'] = resource['extras']['resource_storage_access_method'].toLowerCase();
+				delete resource['extras']['resource_storage_access_method']
```

Comment:
Not a necessary change, but in general, I think this code would be more readable if this pattern were refactored into a function like `renameField(object, oldName, newName, mappingFunction)`.

A call to the function would look like the snippet that follows. This helps leverage function names to document what a handful of lines are actually doing.

```javascript
renameField(
  resource['extras'],
  'resource_storage_access_method', // old name
  'resource_access_method', // new name
  f => f.toLowerCase() // lowercase whatever value exists
);
```